### PR TITLE
[FEATURE] Batched AABB

### PR DESCRIPTION
**Pull request by LeonLiu4**
Adds batched AABB retrieval functionality to enable fast failure detection in robotics training. The new `get_aabb()` methods allow efficient querying of axis-aligned bounding boxes for all entities or specific subsets.

Addresses: #1641

### CODE REVIEW

File: tests/test_rigid_physics.py [hunk 1]
```diff
@@ -2910,3 +2910,46 @@ def test_mesh_primitive_COM(show_viewer, tol):
...
+
+    assert_allclose(torch.tensor(plane_aabb.shape[-1]), torch.tensor(3), atol=tol)
+    assert_allclose(torch.tensor(box_aabb.shape[-1]), torch.tensor(3), atol=tol)
+    assert_allclose(torch.tensor(sphere_aabb.shape[-1]), torch.tensor(3), atol=tol)
+
+    plane_aabb_squeezed = plane_aabb.squeeze() if plane_aabb.ndim == 3 else plane_aabb
+    box_aabb_squeezed = box_aabb.squeeze() if box_aabb.ndim == 3 else box_aabb
+    sphere_aabb_squeezed = sphere_aabb.squeeze() if sphere_aabb.ndim == 3 else sphere_aabb
+
+    assert_allclose(plane_aabb_squeezed, all_aabbs[0], atol=tol)
```
Comment (YilingQiao) [inline tests/test_rigid_physics.py:2953]: Can you also check the actual value of the aabbs for box and sphere since you already know their pos and size?

**Review by YilingQiao — DISMISSED** [review]
